homeassistant/components/unifiprotect/discovery.py

```diff
@@ -0,0 +1,66 @@
+"""The unifiprotect integration discovery."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+from typing import Any
+
+from unifi_discovery import AIOUnifiScanner, UnifiDevice, UnifiService
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DISCOVERY = "discovery"
+DISCOVERY_INTERVAL = timedelta(minutes=60)
+
+
+async def async_start_discovery(hass: HomeAssistant) -> None:
```

Comment:
This could be a callback since we don't await inside.

Reply:
Thanks

https://github.com/home-assistant/core/pull/64413